File: backend/Spotly/Spotly/Models/Kazna.cs
using System;
using System.Collections.Generic;

namespace Spotly.Models;

public partial class Kazna
{
    public int Id { get; set; }

    public string Razlog { get; set; } = null!;

    public string? NovcaniIznos { get; set; }

    public DateTime DatumVrijeme { get; set; }

    public int? AdminId { get; set; }

    public int? KorisnikId { get; set; }

    public int? TipKazneId { get; set; }

    public virtual ICollection<Dokumentacija> Dokumentacijas { get; set; } = new List<Dokumentacija>();

    public virtual Korisnik? Korisnik { get; set; }

    public virtual TipKazne? TipKazne { get; set; }
}

File: frontend/app/src/main/java/org/foi/hr/air/spotly/data/Kazna.kt
package org.foi.hr.air.spotly.data

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable
import java.util.*

@Serializable
data class Kazna(
    @SerialName("id") val id: Int,
    @SerialName("razlog") val razlog: String,
    @SerialName("novcaniIznos") val novcaniIznos: String,
    @SerialName("datumVrijeme") val datumVrijeme: String,
    @SerialName("adminId") val adminId: Int,
    @SerialName("korisnikId") val korisnikId: Int,
    @SerialName("tipKazneId") val tipKazneId: Int
)
